lib/improver/tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -322,6 +322,26 @@ def test_percentiles_non_equal_weights_list(self):
                                            (6, 2, 2))
         self.assertArrayAlmostEqual(result.data, expected_result_array)
 
+    def test_percentiles_different_coordinate_orders(self):
+        """Test the result of the percentile aggregation is the same
+        regardless of the coordinate order in the input cube. Most
+        importantly, the result should be the same regardless of on which
+        side of the collapsing coordinate the percentile coordinate falls."""
+        coord = "time"
+        plugin = WeightedBlendAcrossWholeDimension(coord, 'weighted_mean')
+        weights = None
+        percentile_leading = percentile_cube()
+        time_leading = percentile_cube()
+        time_leading.transpose([1, 0, 2, 3])
```

Comment:
This test does not test whether the percentile aggregation is the same regardless of the the coordinate order of the input cube as the test fails if this line is changed to 
```python
    time_leading.transpose([1, 2, 3, 0])
```

Reply:
It does test this, but you need to reshape the reference data to match the output cube shape. The output cube will have the same order as the input, so the 6 element percentile coordinate will be the final dimension. The below would do what you need and show the behaviour is as expected.

```
expected_result_array = np.reshape(BLENDED_PERCENTILE_DATA1, (6, 2, 2))
expected_result_array = np.moveaxis(expected_result_array, 0, -1)
```